### PR TITLE
DOC-5090: Added a note about passing parameters manually in CLI

### PR DESCRIPTION
**Pull request by AnnaK-R3**

### CODE REVIEW

**Review by yarivus — DISMISSED** [review]
LGTM

File: content/en/platform/corda/5.0-beta/deploying/deployment-tutorials/manual.md [hunk 1]
```diff
@@ -330,6 +334,9 @@ If the schemas are not specified, then the tables will be created in the default
    {{% /tab %}}
    {{< /tabs >}}
 
+  {{< note >}}
+  If you get an error after copying the code from documentation, try passing the exact parameters in the Corda CLI manually.
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-beta/deploying/deployment-tutorials/manual.md:338]:
```suggestion
  If you get an error after copying these commands from the documentation, try passing the exact parameters in the Corda CLI manually.
```

File: content/en/platform/corda/5.0-beta/deploying/deployment-tutorials/manual.md [hunk 2]
```diff
@@ -267,6 +267,10 @@ If the schemas are not specified, then the tables will be created in the default
...
```
Comment (AnnaK-R3) [inline content/en/platform/corda/5.0-beta/deploying/deployment-tutorials/manual.md:271]:
```suggestion
  If you get an error after copying these commands from the documentation, try passing the exact parameters in the Corda CLI manually.
```

**Review by nadinequinr3 — APPROVED** [review]

**Review by yarivus — COMMENTED** [review]
thx @AnnaK-R3

**Pull request merged**